hack/check-everything.sh

```diff
@@ -25,7 +25,7 @@ tmp_root=/tmp
 kb_root_dir=$tmp_root/kubebuilder
 
 # Run verification scripts.
-${hack_dir}/verify.sh
+# ${hack_dir}/verify.sh
```

Comment:
Leftover?

Reply:
Yep, sorry, that sneaked in with the last amend :sweat_smile: